opencti-platform/opencti-graphql/src/database/engine.js

```diff
@@ -3162,7 +3162,9 @@ const elRemoveRelationConnection = async (context, user, elementsImpact) => {
         });
       });
       const bodyUpdate = R.flatten(bodyUpdateRaw);
-      await elBulk({ refresh: true, timeout: BULK_TIMEOUT, body: bodyUpdate });
+      if (bodyUpdate.length > 0) {
```

Comment:
Shouldn't we investigate why we don't have a body rather ?

Reply:
I didn't have time to investigate in details, I guessed the relationships we try to delete have already been deleted. But if you think we should take more time to fix this issue, let me know.

Reply:
It's not about time spent on task it's about knowing what we are fixing.

I'm always afraid of that kind of fix because it might be hidding a real root cause that we will keep having.

Reply:
I understand, but in this code, we have no way of knowing that we will find all the relationships we want to delete in the database. We have the same condition in elIndexElements : 
```
if (bodyUpdate.length > 0) {
  const bulkPromise = elBulk({ refresh: true, timeout: BULK_TIMEOUT, body: bodyUpdate });
  await Promise.all([bulkPromise]);
}
```
What do you think @richard-julien ? did I miss something about this code ?

Reply:
Looks like we have a condition when the list can be empty

```
          if (isEmptyField(fromIndex)) { // No need to clean up the connections if the target is already deleted.
            return updates;
          }
```

If i remember well is a condition to optimize the number of operation to execute, so can be empty in some conditions
Im ok with the fix.